typedargs/basic_structures.py

```diff
@@ -19,10 +26,22 @@ def __iter__(self):
     def __str__(self):
         return str(tuple(self))
 
+    def __repr__(self):
+        return '<{} type_class={}, type_name={}, validators={}, desc={}>' \
+               ''.format(self.__class__.__name__, self.type_class, self.type_name, self.validators, self.desc)
+
 
 class ReturnInfo:
 
     def __init__(self, type_class, type_name, formatter, is_data, desc):
+        """
+        Args:
```

Comment:
Our python styleguide for consistency has init arguments as part of the `class` `docstring`.

Reply:
Also, types should be function annotations

Reply:
done f5acf1206db